### PR TITLE
Billboards type_of: added field to the API and small admin form changes

### PR DESCRIPTION
**Pull request by lightalloy**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
- updated `type_of` radio buttons labels slightly
- removed "For internal use" from `/admin/customization/display_ads/new`
- allowed accessing and setting billboards `type_of` via api
- updated the api docs

## Related Tickets & Documents
- Closes [#534](https://github.com/forem/forem-internal-eng/issues/534)

## QA Instructions, Screenshots, Recordings
- Small changes in the `/admin/customization/display_ads/new` (and edit) forms
изображение
- check the swagger api display_ad docs

### UI accessibility concerns?

_If your PR includes UI changes, please replace this line with details on how
accessibility is impacted and tested. For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?
- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [x] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

### CODE REVIEW

File: spec/requests/api/v1/docs/display_ads_spec.rb
Comment (jaw6) [inline spec/requests/api/v1/docs/display_ads_spec.rb:62]: There's a minor typo here:

```suggestion
                         Types of the billboards:
```

**Review by jaw6 — APPROVED** [review]

**Comment by jaw6** [comment]
There will likely be some merge conflicts between this and #19126, but this looks likely to merge first, I can update #19126.

**Comment by mirie** [comment]
@lightalloy this is approved, can this be merged in tomorrow? For any PRs that are OK to merge by anyone on the team once approved, please use the label `merge by any core`. Thanks!

**Pull request merged**
